agent/api/task/task_test.go

```diff
@@ -927,6 +928,14 @@ func TestTaskFromACS(t *testing.T) {
 					HostConfig: strptr("hostconfig json"),
 					Version:    strptr("version string"),
 				},
+				Secrets: []*ecsacs.Secret{
+					{
+						Name:      strptr("secret"),
+						ValueFrom: strptr("/test/secret"),
+						Provider:  strptr("ssm"),
```

Comment:
did you mean to set these to `asm`?

Reply:
This test is to test the TastFromASM function in task.go. I just want to make sure secrets can be unmarshalled correctly. So no need to be asm provider.